src/ExtensionsSample/Samples/EasyTableSamples.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ExtensionsSample.Models;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.EasyTables;
+using Microsoft.WindowsAzure.MobileServices;
+
+namespace ExtensionsSample
+{
+    public static class EasyTableSamples
+    {
+        // POCO output binding 
+        //   The binding inserts the newly created item into the Easy Table when the 
+        //   method successfully exits.         
+        public static void InsertItem([EasyTable] out Item newItem)
```

Comment:
Well it was in Program.cs: `            host.Call(typeof(EasyTableSamples).GetMethod("InsertItem"));
`. But without UseEasyTables() I think it'll blow up when you hit F5. I think I'll add it back and comment it out with a note about how to 'turn on' EasyTables.